### PR TITLE
New features for EF Core 5.0 preview 6

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

**Comment by ajcvickers** [comment]
@smitpatel @maumar Can you take a look at the query parts?
@roji Does the wording around consistency seem okay?

File: entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md [hunk 1]
```diff
@@ -8,21 +8,315 @@ uid: core/what-is-new/ef-core-5.0/whatsnew
...
+> Unhandled exception. System.InvalidOperationException: The LINQ expression 'DbSet<Artist>()
+>      .Where(a => a.Name.Equals(
+>          value: "The Unicorns", 
+>          comparisonType: CurrentCulture))' could not be translated. Additional information: Translation of 'string.Equals' method which takes 'StringComparison' argument is not supported. See https://go.microsoft.com/fwlink/?linkid=2129535 for more information. Either rewrite the query in a form that can be translated, or switch to client evaluation explicitly by inserting a call to either AsEnumerable(), AsAsyncEnumerable(), ToList(), or ToListAsync(). See https://go.microsoft.com/fwlink/?linkid=2101038 for more information.
+
+### Specify transaction ID
+
+This feature was contributed from the community by [@Marusyk](https://github.com/Marusyk). Many thanks for the contribution!
+
+EF Core exposes a transaction ID for correlation of transactions across calls. This is typically set by EF Core when a transaction is started. If the application starts the transaction instead, then this feature allows the application to explicitly set the transaction ID so it is correlated correctly everywhere it is used. For example:
```
[the remark this answers is not in the document]
Comment (roji) [reply entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md:190]: (very nitpicky, very ignorable)

[the remark this answers is not in the document]
Comment (smitpatel) [reply entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md:190]:
```suggestion
EF Core exposes a transaction ID for correlation of transactions across calls. This ID is typically set by EF Core when a transaction is started. If the application starts the transaction instead, then this feature allows the application to explicitly set the transaction ID so it is correlated correctly everywhere it is used. For example:
```
😅

[the remark this answers is not in the document]
Comment (smitpatel) [reply entity-framework/core/what-is-new/ef-core-5.0/whatsnew.md:190]: (In addition to whatever @roji said)

**Pull request merged**